### PR TITLE
Updated readme.md

### PR DESCRIPTION
**Pull request by trishahanlon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,38 @@
...
+   - use the `--set` flag and pass configuration from the command line.
+- Use the `--set-file` flag to set individual values from a file when the value itself is too long for the command line or is dynamically generated.
+- To force a string value use `--set-string`. 
 
-```console
-helm install -f myvalues.yaml my-agent optimizely-agent/agent
+For example:
+
+```shell
+helm install --values myvalues.yaml my-agent optimizely-agent/agent
```
Comment (msohailhussain) [inline README.md]: Please specify version number, otherwise it will cause problem or specify if it's causing issue then specify version number.

**Review by msohailhussain — DISMISSED** [review]
lgtm. just one comment.

File: README.md (hunk 1 above)
Comment (trishahanlon) [reply README.md]: Updated in [commit](https://github.com/optimizely/optimizely-agent-helm/pull/12/commits/89ef4899b91b3909e294f793951afd508ce87727)

**Review by msohailhussain — COMMENTED** [review]
lgtm

**Review by msohailhussain — APPROVED** [review]

**Pull request merged**